Detailed configuration of the generator (#85)
victools/jsonschema-generator: jsonschema-maven-plugin/README.md

<plugin>
    <groupId>com.github.victools</groupId>
    <artifactId>jsonschema-maven-plugin</artifactId>
    <executions>
        <execution>
            <goals>

>                 <goal>generator-schema</goal>

Author: Correct. I'll fix.

`DRAFT_7` is the default, if not specified.

### Configuring the generator
The JSON schema generator can be configured with the options as they are defined in `com.github.victools.jsonschema.generator.Option` 
The following example shows the possible elements:
```xml
<configuration>
    <className>com.myOrg.myApp.MyClass</className>
    <options>
        <preset>FULL_DOCUMENTATION</preset>
        <enabled>
            <option>DEFINITIONS_FOR_ALL_OBJECTS</option>
            <option>FORBIDDEN_ADDITIONAL_PROPERTIES_BY_DEFAULT</option>
        </enabled>
        <disabled>SIMPLIFIED_ENUMS</disabled>
    </options>
</configuration>
```

> The `preset` element can select one of the values from `com.github.victools.jsonschema.generator.Option`.
> In case no prefix is not specified `PLAIN_JSON` is taken as default value.
> 
> Additional to the prefix a number of options that will be enabled and disabled can be specified.
> The syntax supports a single value, and multiple values as nested elements.

Reviewer: ```suggestion
The `preset` element can select one of the values from `com.github.victools.jsonschema.generator.Option`.
In case no `preset` is specified, `PLAIN_JSON` is taken as default value.

Additional to the `preset` a number of options can be either enabled and/or disabled.
The syntax supports a single value as well as multiple values as nested elements.
```

Author: Fixed.
